xml/obs_basic_workflow.xml

```diff
@@ -312,6 +313,35 @@ Fri Aug 23 08:42:42 UTC 2017 - &exampleuser_mail;</screen>
     <screen>&prompt.user;<command>osc</command> commit</screen>
    </step>
   </procedure>
+  <para>
+   To delete a file in your working directory, merely deleting it from the local
+   filesystem ("rm FILENAME") is probably not going to be sufficient, since
```

Comment:
Probably? You know for sure that is will not be sufficient once it's version controlled :)

Reply:
You're right: version control has no concept of sarcasm ;-)